tools/obscuroscan_v2/frontend/src/lib/config.dev.js

```diff
@@ -1,5 +1,5 @@
 class Config {
-    static backendServerAddress = "http://127.0.0.1:8080"
+    static backendServerAddress = "http://127.0.0.1:43910"
```

Comment:
What's this port about? You need to configure that somewhere locally to run dev server or it just works?

Reply:
It's a placeholder. It should point to the api port for the dev environment. I guess I'll change, and set it as default, once I hook up the launcher to obscuroscan ?